launcher/ui/widgets/MinecraftSettingsWidget.cpp

```diff
@@ -50,13 +50,11 @@ MinecraftSettingsWidget::MinecraftSettingsWidget(MinecraftInstancePtr instance,
     m_ui->setupUi(this);
 
     if (m_instance == nullptr) {
-        for (int i = 0; i < m_ui->settingsTabs->count(); ++i) {
+        for (int i = m_ui->settingsTabs->count() - 1; i > 0; --i) {
```

Comment:
You may want to make it >= than 0:
```suggestion
        for (int i = m_ui->settingsTabs->count() - 1; i >= 0; --i) {
```

Reply:
No? Then it won't take into account index 0

Reply:
oh wait why is my brain not working